contracts/src/TeleporterTokenDestination.sol

```diff
@@ -53,6 +53,20 @@ abstract contract TeleporterTokenDestination is
     /// @notice The address of the source token bridge instance this contract receives tokens from.
     address public immutable tokenSourceAddress;
 
+    /**
+     * @notice The number of decimal places in the denomination of the source
+     * token.
+     * @dev Used to derive tokenMultiplier and multiplyOnDestination.
+     */
+    uint8 public immutable sourceTokenDecimals;
+
+    /**
+     * @notice The number of decimal places in the denomination of the
+     * destination token.
+     * @dev Used to derive tokenMultiplier and multiplyOnDestination.
+     */
+    uint8 public immutable tokenDecimals;
+
```

Comment:
can you update the `tokenMultiplier` comment for outdated `decimalsShift` reference?

Reply:
addressed in https://github.com/ava-labs/teleporter-token-bridge/pull/142/commits/82b9d7c0aac6ea8a12ab43f9d1e4ab9a27fb2637